include/torch-mlir/Conversion/TorchOnnxToTorch/OnnxLstmExpander.h

```diff
@@ -0,0 +1,14 @@
+#include "torch-mlir/Conversion/TorchOnnxToTorch/Patterns.h"
+#include "torch-mlir/Conversion/TorchOnnxToTorch/Utils.h"
+#include "torch-mlir/Dialect/Torch/IR/TorchOps.h"
+#include "torch-mlir/Dialect/Torch/Utils/Utils.h"
+
+// debug
+#include "llvm/Support/Debug.h"
+#define DEBUG_TYPE "torch-onnx-to-torch-patterns"
```

Comment:
Include a `using mlir::torch::Torch` and `using mlir`. It can let you avoid using all of the namespace qualifiers.